kube/services/revproxy/nginx.conf

```diff
@@ -247,7 +247,7 @@ server {
   more_set_headers "Access-Control-Expose-Headers: Content-Length,Content-Range";
 
   # update service release cookie
-  add_header Set-Cookie "service_releases=${service_releases};Path=/;Max-Age=600;HttpOnly;Secure;SameSite=Lax";
+  # add_header Set-Cookie "service_releases=${service_releases};Path=/;Max-Age=600;HttpOnly;Secure;SameSite=Lax";
```

Comment:
is this intentional? I thought this is to add some madatory security related cookies

Reply:
This is/ was used for some canary deployments, which was never fully implemented. We could keep it in, but I don't think we need it.